{"name": "Login with invalid test data", "status": "passed", "description": "This test of login with invalid data", "steps": [{"name": "opening main page", "status": "passed", "start": 1666940385760, "stop": 1666940391856}, {"name": "Login with username and password", "status": "passed", "parameters": [{"name": "username", "value": "'Admin'"}, {"name": "password", "value": "'dmin123'"}], "start": 1666940391856, "stop": 1666940392886}], "attachments": [{"name": "log", "source": "f778ce9e-2f1f-43fa-80be-5d439dcce66e-attachment.txt", "type": "text/plain"}], "start": 1666940385760, "stop": 1666940396328, "uuid": "b456a84e-8042-4b00-a495-1abc3b711330", "historyId": "7111698c25d44dabac4f20fcce42c3f7", "testCaseId": "6cd70974c8fc663283d871c2e92f9648", "fullName": "tests.test_Login_02.Test_login_02#test_login", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_Login_02"}, {"name": "subSuite", "value": "Test_login_02"}, {"name": "host", "value": "DESKTOP-SQ8UVGQ"}, {"name": "thread", "value": "10596-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_Login_02"}]}